make_release/this_week_in_nu_weekly.nu

```diff
@@ -52,12 +52,17 @@ def query-week-span [] {
     }
 }
 
-# 2019-08-23 was the release of 0.2.0, the first public release
-let week_num = ((seq date -b '2019-08-23' -n 7 | length) - 1)
-print $"# This week in Nushell #($week_num)(char nl)"
-
-if ($env | select GITHUB_USERNAME | is-empty) or ($env | select GITHUB_PASSWORD | is-empty) {
-    print 'Please set GITHUB_USERNAME and GITHUB_PASSWORD in $env to use this script'
-} else {
-    query-week-span
+def main [
```

Comment:
I'm not sure it's important that these all be modules, it should work if you just do `source script.nu`